packages/augur-ui/src/services/initialize.ts

```diff
@@ -6,25 +6,18 @@ import { windowRef } from "utils/window-ref";
 import getInjectedWeb3Accounts from "utils/get-injected-web3-accounts";
 
 export const connect = async (env: EnvObject, callback: NodeStyleCallback = logError) => {
-
-  let provider = new JsonRpcProvider(env["ethereum-node"].http);
-  let isWeb3 = false;
-  let account = "";
-
-  const bootstrap = async (provider, account, isWeb3) => {
-    await augurSdk.makeApi(provider, account, provider.getSigner(), env, isWeb3);
-  };
-
   const injectedAccount = await getInjectedWeb3Accounts();
   const loggedInAccount = windowRef.localStorage.getItem("loggedInAccount");
 
   // Use injected provider if returning User has a unlocked injected account that matches the current logged in account
   if (injectedAccount && (loggedInAccount === injectedAccount[0])) {
-    provider = new Web3Provider(windowRef.web3.currentProvider);
-    account = windowRef.web3.currentProvider.selectedAddress,
-    isWeb3 = true;
+    const provider = new Web3Provider(windowRef.web3.currentProvider);
+    const account = windowRef.web3.currentProvider.selectedAddress;
+
+    await augurSdk.makeApi(provider, account, provider.getSigner(), env, false);
```

Comment:
question... what is the isWeb3 parameter suppose to denote?

I thought it was to indicate providers that were injected, vs providers we need to bootstrap (hardware wallets, non-logged-in users)

If that correct? If so, then line 17 should pass in TRUE and line 19 should pass in false.

Reply:
You are right!